torch/_export/__init__.py

```diff
@@ -338,14 +343,20 @@ def export(
         loss_output=graph_signature.backward_signature.loss_output
     ) if graph_signature.backward_signature is not None else None
 
+    def to_str_list(sig_component: List[Any]):
+        return [str(v) for v in sig_component]
+
+    def to_str_dict(sig_component: Dict[Any, Any]):
+        return [str(k): str(v) for (k, v) in sig_component]
+
     export_graph_signature = ExportGraphSignature(
-        parameters=graph_signature.parameters,
-        buffers=graph_signature.buffers,
-        user_inputs=graph_signature.user_inputs,
-        user_outputs=graph_signature.user_outputs,
-        inputs_to_parameters=graph_signature.inputs_to_parameters,
-        inputs_to_buffers=graph_signature.inputs_to_buffers,
-        buffers_to_mutate=graph_signature.buffers_to_mutate,
+        parameters=to_str_list(graph_signature.parameters),
```

Comment:
What's the `to_str_list` / `to_str_dict` doing? Why is it necessary? Does it need to be parsed back to non-str objects?

Reply:
The original types are aot autograd internal types that just alias str. I am not comfortable exposing internal types that can change on our public interface. 

They don't need to be converted back I think